src/product/ProductInfo.jsx

```diff
@@ -1,27 +1,27 @@
 import React from 'react';
-import { ProductRating } from './ProductRating';
-import { ProductDetails } from './ProductDetails';
-import { ProductDescription } from './ProductDescription';
+import {ProductRating} from './ProductRating';
+import {ProductDetails} from './ProductDetails';
+import {ProductDescription} from './ProductDescription';
 import './ProductInfo.css'
 
 export const ProductInfo = props => {
-    const { runtime, releaseDate, overview, title, voteAverage, tagline} = props;
+    const {movie} = props;
     return (
-         <div className='product-info'>
-             <div className='product-info-title-rating'>
-                 <span className='product-info-title'>{ title }</span>
-                 <ProductRating
-                 rating = { voteAverage }
-                 />
-             </div>
-            <div className='product-info-tagline'>{ tagline }</div>
-             <ProductDetails
-             runtime = { runtime }
-             releaseDate = { releaseDate }
-             />
-             <ProductDescription
-             overview = { overview }
-             />
-         </div>
+        <div className='product-info'>
+            <div className='product-info-title-rating'>
+                <span className='product-info-title'>{movie.title}</span>
+                <ProductRating
+                    rating={movie.vote_average}
```

Comment:
we can use desctructuring to make jsx more readable: `const { movie: { a, b, c } } = props`